target/linux/ath79/image/generic.mk

```diff
@@ -1398,6 +1398,16 @@ define Device/jjplus_ja76pf2
 endef
 TARGET_DEVICES += jjplus_ja76pf2
 
+define Device/jjplus_jwap230
+  SOC := qca9558
+  DEVICE_VENDOR := jjPlus
+  DEVICE_MODEL := JWAP230
+  KERNEL := kernel-bin | append-dtb | lzma | uImage lzma
```

Comment:
This is the same as in Device/Default and thus can be dropped ...

Reply:
I switched to the default sysupgrade rule. In the end, the image definition is rather minimalistic now.